code/modules/clothing/head/jobs.dm

```diff
@@ -1,3 +1,5 @@
+/obj/item/clothing/head/

```

Comment:
Move this variable to the class definition [here](https://github.com/tgstation/tgstation/blob/2da8867f67a669509b40055a404047877afb5b00/code/modules/clothing/clothing.dm#L269)